frontend/app/model/Benefits.scala

```diff
@@ -58,57 +55,64 @@ object Benefits {
 
   val friendBenefitsList = benefitsFilter(
     "book_tickets",
-    "digital_digest",
-    "video_highlights"
+    "offers_competitions",
```

Comment:
Relying on strings might cause issues when typos occur. On the other hand this is contained in the same file, and the risks are small.